gapis/resolve/memory.go

```diff
@@ -181,6 +182,46 @@ func pNextPrecision(typeIndex uint64) (int, error) {
 	}
 }
 
+// expandCharArrays goes through the range and attaches a human-readable representation to char arrays/slices
+func expandCharArrays(ctx context.Context, ranges []*service.TypedMemoryRange) ([]*service.TypedMemoryRange, error) {
+	if len(ranges) == 0 {
+		return ranges, nil
+	}
+
+	for _, rng := range ranges {
+		ty, err := types.GetType(rng.Type.TypeIndex)
```

Comment:
I know the go-ism is to use short / single letter names for local variables, but we've found this a real obstacle to reading the gapis code in the past. Melih has made an effort to expand the names of variables to be more descriptive in the past, so it would be great if we could continue that trend here with new code. Perhaps type instead of ty or range instead of rng. This code is neat and much more readable than the gapis average, but it's still good to continue with the descriptive names to reenforce the trend.

Reply:
In this case, `type` is a keyword in Go, therefore we cannot use it. The usual word I see in the codebase is `typ`

Reply:
Done, opted for more descriptive names `memType` and `typedRange`